### ISSUE TITLE
Access to calendar object from DOM element

### ISSUE THREAD

**Issue by bryant-essensepartners** [fullcalendar/fullcalendar#6861]
### Checklist

Please mark these items with an [x]

  - [x] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [x] Other people will find this feature useful

### Feature Description

I've worked with FullCalendar various times over the last couple of years, its my go to calendar JS library for projects because of its versatility! In many of my projects I have used the 'getCalendar' method after the calendar has already been initialized. There's been various cases where I didn't have direct access to the declaration and initialization process of the Calendar (so I couldn't save the FullCalendar object to a variable) and the 'getCalendar' method allowed me to access the FullCalendar JS object and apply various customizable features. I just ran into another issue where a FullCalendar instance was created and I want to associate an event to that FullCalendar object but have no way of accessing the instance. 

In short, I am kindly requesting that you bring the following method back (code taken from v3) or something similar:

// a convenient utility for getting the calendar object.
// you can call methods on the calendar object directly.
var calendar = $('#calendar').fullCalendar('getCalendar'); 

I know other developers and I will benefit from this feature as it introduces a versatile approach to getting the full calendar instance and removes the limitations of having to keep record of the instances via variables. 

Thank you!

**Comment by lionelweber** [comment]
I've solved that just by attaching the calendar object to a data attribute (still using JQuery):
```
var calendarEl = $('#calendar').get(0); //imagine your calendar is on a div with id calendar
var calendar = new FullCalendar.Calendar(calendarEl,{}); //instanciate your callendar
$(calendarEl).data('fullCalendarObj',calendar); //save the calendar pointer in data attached to Dom object
```
Then elswhere in my code I can access to the object instance with:
```
var calendar = $('#calendar').data('fullCalendarObj'); //retreive the object from data attr
if(calendar && calendar.isRendered){ //do code based on object, in this example change the height if rendered
	calendar.setOption('height',$('#calendar').height()-3);
}
```

**Comment by mcolegro** [comment]
@lionelweber Thanks for posting your solution.  I've been looking for a way to access the calendar instance for about 2 weeks now.  All I needed to do was refresh the calendar when the tab it was on was clicked, but I could not figure out how to get access to the calendar object.  Attaching the object to the data attribute worked for my need.

Is there a way to do the same thing in plain vanilla JavaScript?

**Comment by arshaw** [comment]
Some help:
https://stackoverflow.com/questions/66816366/full-calendar-implemented-using-jquery

Unofficial jquery connector for more recent versions:
https://github.com/fullcalendar/fullcalendar/blob/master/examples/_jquery-connector.html

Ticket to officially revive jQuery support:
https://github.com/fullcalendar/fullcalendar/issues/4562

**Comment by tobiasnitsche** [comment]
@arshaw why is this issue closed? It would be actually really useful to me too and i dont like to install jquery for that.

**Comment by toomanylogins** [comment]
I agree, I am loading content in modals via ajax calls and cant work out how to access the calendar object from the div element. The docs are very poor.
